src/reflect/scala/reflect/api/Trees.scala

```diff
@@ -802,13 +802,13 @@ trait Trees { self: Universe =>
    *    import expr.{selectors}
    *
    *  Selectors are a list of ImportSelectors, which conceptually are pairs of names (from, to).
-   *  The last (and maybe only name) may be a nme.WILDCARD. For instance:
+   *  The last (and maybe only name) may be a nme.EMPTY. For instance:
    *
-   *    import qual.{x, y => z, _}
+   *    import qual.{w => _, x, y => z, _}
    *
    *  Would be represented as:
    *
-   *    Import(qual, List(("x", "x"), ("y", "z"), (WILDCARD, null)))
+   *    Import(qual, List(("w", EMPTY), ("x", "x"), ("y", "z"), (EMPTY, EMPTY)))
```

Comment:
As mentioned in the discussion, could we use `(WILDCARD, WILDCARD)` for the last element?

Reply:
Now (WILD, WILD) means import backticked underscore, possibly with rename. Otherwise, the encoding is unchanged.